.github/workflows/ci.yml

```diff
@@ -57,18 +62,30 @@ jobs:
           toit-version: ${{ steps.constants.outputs.toitVersion }}
 
       - name: Setup Qemu
-        # TODO(florian): Enable again when Windows tests work again.
-        # TODO(florian): Enable again on macOS once qemu is building again.
-        if: runner.os == 'Linux'
         shell: bash
         run: |
           mkdir -p downloads
           cd downloads
-          curl -L  -o qemu-${{ runner.os }}.tar.gz https://github.com/toitlang/qemu/releases/download/v0.0.3/qemu-${{ runner.os }}.tar.gz
-          tar x -f qemu-${{ runner.os }}.tar.gz
-          echo "QEMU_EXEC=${{ github.workspace }}/downloads/qemu-${{ runner.os }}/qemu-system-xtensa$BIN_EXTENSION" >> $GITHUB_ENV
+          if [[ ${{ runner.os }} == "Linux" ]]; then
+            curl -L -o qemu-linux.tar.xz $QEMU_LINUX_URL
+            tar x -f qemu-linux.tar.xz
+            echo "QEMU_EXEC=$PWD/qemu/bin/qemu-system-xtensa" >> $GITHUB_ENV
+          elif [[ ${{ runner.os }} == "macOS" ]]; then
+            curl -L -o qemu-macos.tar.xz $QEMU_MACOS_URL
+            tar x -f qemu-macos.tar.xz
+            echo "QEMU_EXEC=$PWD/qemu/bin/qemu-system-xtensa" >> $GITHUB_ENV
+          elif [[ ${{ runner.os }} == "Windows" ]]; then
+            curl -L -o qemu-windows.tar.xz $QEMU_WINDOWS_URL
+            tar x -f qemu-windows.tar.xz
+            echo "QEMU_EXEC=$PWD/qemu/bin/qemu-system-xtensa.exe" >> $GITHUB_ENV
+          fi
           ls $QEMU_EXEC
 
+      - name: Install qemu dependency
```

Comment:
```suggestion
      - name: Install Qemu dependency
```

Reply:
obsolete. I moved the installation into the other dependency installation step.